lib/game.rb

```diff
@@ -0,0 +1,124 @@
+require "./lib/card.rb"
+require "./lib/deck.rb"
+require "./lib/player.rb"
+require "./lib/turn.rb"
+
+class Game
+    attr_reader :full_deck, :player1, :player2, :cards, :deck1, :deck2
+
+def initialize(turn)
+  @full_deck = full_deck
+  @deck1 = @full_deck[0, 26]
+  @deck2 = @full_deck[26, 26]
+  @player1 = Player.new("Perrin", @deck1) #Player.new("Perrin", deck1)
+  @player2 = Player.new("Egwene", @deck2) #Player.new("Egwene", deck2)
+  @cards = cards
+  @current_turn = current_turn
+end
```

Comment:
In your `#initialize` method, I couldn't find where `full_deck` on line 10, `cards` on line 15, and `current_turn` on line 16 were coming from.

Here is how I refactored this method:

```ruby
def initialize
  @full_deck = create_deck #<== calling the 'create_deck' method to get my cards
  @deck1 = @full_deck[0, 26]
  @deck2 = @full_deck[25, 26]
  @player1 = Player.new("Perrin", @deck1)
  @player2 = Player.new("Perrin", @deck2)
end
```

You'll notice that I removed `@cards = cards` and `@current_turn = current_turn` because I didn't see a `cards` or `current_turn` method inside this class.